server/server.js

```diff
@@ -4,20 +4,23 @@ const bodyParser = require('body-parser');
 const passport = require('passport');
 const redis = require('connect-redis')(session);
 
-const routes = {
-  board : require('./routes/board')
-}
+const routesBoard = require('./routes/board');
 
+// To do: find alternative for defaults
 const PORT = process.env.PORT || 8080;
+const REDIS_HOST_PORT = process.env.REDIS_HOST_PORT || 6379;
 const ENV = process.env.development || 'development';
 const SESSION_SECRET = process.env.SESSION_SECRET || 'keyboard cat';
 
 const app = express();
 
-app.use(bodyParser.urlencoded({ extended: false }));
+if (!REDIS_HOST_PORT) {
+  throw new Error('REDIS_HOST_PORT not set');
+}
+
 app.use(bodyParser.json());
 app.use(session({
-  store : new redis({ url : 'redis://redis-server:6379', logErrors : true }),
+  store : new redis({ url : `redis://redis-server:${REDIS_HOST_PORT}`, logErrors : true }),
```

Comment:
Both the REDIS_HOST and REDIS_HOST_PORT should be environment variables. In this case your solution is fixed to a single host address.